src/modules/organization/InternalGroups.tsx

```diff
@@ -32,75 +25,66 @@ const InternalGroupsContainerTitle = styled.h2`
 const InternalGroupsContainer = styled.div`
   display: grid;
   grid-template-columns: repeat(3, 1fr);
-  ${props => props.theme.media.mobile} {
-    grid-template-columns: 1fr;
+  ${props => props.theme.media.largeScreen} {
+    grid-template-columns: 1fr 1fr;
   }
-  ${props => props.theme.media.smallScreen} {
+  ${props => props.theme.media.mobile} {
     grid-template-columns: 1fr;
   }
   gap: 10px;
   border-radius: 4px;
+  margin-bottom: 1.618em;
 `
 
 interface AllInternalGroupsReturns {
-  allInternalWorkGroups: InternalGroupNode[]
-}
-
-interface AllInterestGroupsReturns {
-  allInterestGroups: InternalGroupNode[]
+  internalGroups: InternalGroupNode[]
+  interestGroups: InternalGroupNode[]
 }
 
 export const InternalGroups: React.VFC = () => {
   const history = useHistory()
-  const {
-    loading: internalLoading,
-    error: internalError,
-    data: internalData,
-  } = useQuery<AllInternalGroupsReturns>(ALL_INTERNAL_WORK_GROUPS)
-  const {
-    loading: interestLoading,
-    error: interestError,
-    data: interestData,
-  } = useQuery<AllInterestGroupsReturns>(ALL_INTEREST_GROUPS)
 
-  if (internalError) return <FullPageError />
-  if (internalLoading || !internalData) return <FullContentLoader />
+  const allInternalGroupTypesQuery = useQuery<AllInternalGroupsReturns>(
```

Comment:
destructure as you do in other queries
```suggestion
  const { loading, error, data } = useQuery<AllInternalGroupsReturns>(
```

Reply:
Yep, changed implementation and destructured the query.<img src="https://media2.giphy.com/media/KPn24x701Asus/giphy.gif"/>